src/main/java/org/seqdoop/hadoop_bam/util/VCFFileMerger.java

```diff
@@ -85,11 +90,9 @@ private static boolean writeHeader(OutputStream out, Path outputPath, List<Path>
     boolean blockCompressed = isBlockCompressed(parts);
     boolean bgzExtension = outputPath.toString().endsWith(BGZFCodec.DEFAULT_EXTENSION);
     if (blockCompressed && !bgzExtension) {
-      System.err.println("WARNING: parts are block compressed, but output does not " +
-          "have .bgz extension: " + outputPath);
+      logger.warn("Parts are block compressed, but output does not have .bgz extension: {}", outputPath);
     } else if (!blockCompressed && bgzExtension) {
-      System.err.println("WARNING: parts are not block compressed, but output has a " +
-          ".bgz extension: " + outputPath);
+      logger.warn("WARNING: parts are not block compressed, but output has a .bgz extension: {}", outputPath);
```

Comment:
&rarr; `logger.warn("Output has a .bgz extension, but parts are not block compressed: {}", outputPath);`